### DISCUSSION TITLE
Database warning after upgrade?!

### DISCUSSION THREAD

**Discussion by johndoe0815** [TandoorRecipes/recipes#2754] [Q&A]
Hi,
I just updated from 1.5.6 to 1.5.8.

Under 1.5.6, the system page shows:
![grafik](https://github.com/TandoorRecipes/recipes/assets/92877294/fecab784-0be2-44c5-8509-b937281efb00)

Without changing the docker compose file but just by redeploying the stack in portainer to update, afterwards the system page shows
![grafik](https://github.com/TandoorRecipes/recipes/assets/92877294/d7f33064-540b-448a-9cd6-9ac0cacf70fe)

Is it correct that Sqlite is now true and that I get a warning about that?

By the way, this whole page does still not show the actual version number of tandoor that I am on, just
![grafik](https://github.com/TandoorRecipes/recipes/assets/92877294/58b46cca-63f4-477c-ac29-274004c38f46)

**Comment by smilerz** [comment]
Duplicate #2752
